packages/app-config-writer/README.md

```diff
@@ -0,0 +1,41 @@
+# @sap-ux/app-config-writer
+
+The `@sap-ux/app-config-writer` adds smartlinks configuration for a SAP UX project. 
```

Comment:
`app-config-writer` sounds like a more generic component than something that adds `smartlinks` to a project.
If the intention is to add more functionality here in the future, then please write a few words about what the app-config-writer is supposed to be in the future.

Please also add a few words about what a smartlink is and what situation, a Fiori developer would need/use them

Reply:
Thanks, added a few words to the general app-config-writer and a small description for the smartlinks configuration - would that work?

Reply:
Hi @tobiasqueck, would the description work like this? Was the only open point still to be resolved.